src/components/v2/Button/FormikSubmitButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+import { PrimaryButton, IButtonProps } from '.';
+
+interface IFormikSubmitButtonProps extends IButtonProps {
+  disabled?: boolean;
+  loading: boolean;
+  disabledLabel?: string;
+  enabledLabel: string;
+}
+
+export const FormikSubmitButton = ({
+  disabledLabel,
+  enabledLabel,
+  variant = 'primary',
+  disabled = false,
+  loading = false,
```

Comment:
Since `loading` is already defined in `IButtonProps` and a default value is assigned to it in the `Button` component, you can remove it here.